daemon/oci_linux_test.go

```diff
@@ -35,7 +35,7 @@ func TestTmpfsDevShmNoDupMount(t *testing.T) {
 		},
 	}
 
-	// Mimick the code flow of daemon.createSpec(), enough to reproduce the issue
+	// Mimic the code flow of daemon.createSpec(), enough to reproduce the issue
```

Comment:
Looks like both are ok (didn't know that 😅 ) https://en.wiktionary.org/wiki/mimick